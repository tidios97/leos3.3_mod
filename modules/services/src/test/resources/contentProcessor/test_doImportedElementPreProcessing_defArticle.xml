<article xmlns="http://docs.oasis-open.org/legaldocml/ns/akn/3.0" xml:id="_imp_art_d1e5242_6jPqhP">
    <num>Article 64</num>
    <heading>Definitions</heading>
    <paragraph>
        <num/>
        <subparagraph>
            <content>
                <p>par 1</p>
            </content>
        </subparagraph>
        <list>
            <point>
                <num>(1)</num>
                <content>
                    <p>This should be (1)</p>
                </content>
            </point>
            <point>
                <num>(2)</num>
                <content>
                    <p>This should be (2)</p>
                </content>
            </point>
        </list>
    </paragraph>
</article>
