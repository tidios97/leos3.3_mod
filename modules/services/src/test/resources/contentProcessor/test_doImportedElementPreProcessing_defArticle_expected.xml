<article xmlns="http://docs.oasis-open.org/legaldocml/ns/akn/3.0" refersTo="~_ART_DEF" xml:id="_imp__imp_art_d1e5242_6jPqhP_Yoh0kL">
    <num>Article 64</num>
    <heading>Definitions</heading>
    <paragraph>
        <num></num>
        <subparagraph>
            <content>
                <p>par 1</p>
            </content>
        </subparagraph>
        <list>
            <point>
                <num>(1)</num>
                <content>
                    <p>This should be (1)</p>
                </content>
            </point>
            <point>
                <num>(2)</num>
                <content>
                    <p>This should be (2)</p>
                </content>
            </point>
        </list>
    </paragraph>
</article>
